app/models/data/local_authority_user.rb

```diff
@@ -18,7 +18,7 @@ def generate_csv
 
   private
 
-    # @param users [ActiveRecord::Relation<User>]
+    # @param users [ActiveRecord::Relation]
```

Comment:
There is no incoming argument to the method. Run `yard` and see what it says. Hopefully something.

Reply:
Sorry, don't know why/how I did that [5b915ad](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/636/commits/5b915add25929571f1c77953ffb21a33db3236fd)